### DISCUSSION TITLE
When can I see the FLET studio?

### DISCUSSION THREAD

**Discussion by 7576457** [flet-dev/flet#1920] [General]
I am in my last year of university, and I need to write a diploma work. I really want to create a mobile app in Flet Studio. There was information in the blog that it was planned to be released at the end of this year, but I did not see the date of creation of the record. Tell me, can I confidently expect the appearance of Flet Studio before the end of this year? This is very exciting for me

**Comment by SKbarbon** [comment]
Hi @Klu1d ,
Flet Studio is been published a time ago. The app name is "flet" only and not "Flet Studio".

To install it on iOS:
[flet studio - ios](https://apps.apple.com/app/id1624979699)

To install it for Android:
[flet studio- android](https://play.google.com/store/apps/details?id=com.appveyor.flet)
